theano/gpuarray/linalg.py

```diff
@@ -407,6 +407,15 @@ def make_node(self, A):
                                 [GpuArrayType(A.dtype, broadcastable=[False],
                                               context_name=ctx_name)()])
 
+    def prepare_node(self, node, storage_map, compute_map, impl):
```

Comment:
Yes, I tested in (running nosetests on `test_linalg.py`), assertion is evaluated and succeeds.